src/App.tsx

```diff
@@ -19,55 +55,36 @@ export const App: React.FC = () => {
               placeholder="Enter a part of the name"
               className="input"
               data-cy="search-input"
-            />
-          </div>
+              value={selectedPerson?.name || query}
+              onChange={handleQueryChange}
+              onFocus={() => setIsVisible(true)}
+              onBlur={() => setIsVisible(false)}
 
-          <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
-            <div className="dropdown-content">
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Bernard Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Antone Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Elisabeth Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter de Decker</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Petronella de Decker</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Elisabeth Hercke</p>
-              </div>
-            </div>
+            />
           </div>
-        </div>
+          {isVisible && (
```

Comment:
You shouldn't display dropdown if filteredPeople array is empty
<img width="416" alt="Screenshot 2024-03-18 at 18 20 33" src="https://github.com/mate-academy/react_autocomplete/assets/117587463/27ebbc13-ef57-47b3-97d0-e6957623791c">